core/amber/src/main/python/pytexera/udf/examples/count_batch_operator.py

```diff
@@ -0,0 +1,16 @@
+from pytexera import *
+
+
+class CountBatchOperator(UDFBatchOperator):
+    BATCH_SIZE = 10
+
+    def __init__(self, *args):
+        if len(args) >= 1:
+            self.BATCH_SIZE = args[0]
+        super().__init__()
+        self.count = 0
```

Comment:
as we are providing this operator as an example to the users as well, can we make the code as standard as possible? here taking args and reassigning the self.BATCH_SIZE is NOT what we want the users to follow.

To properly test "initialize" and "update" an operator, the current way of mocking an operator instance is not good enough. you will need to mock the control messages`InitializeOperatorLogic` and `UpdateOperatorLogic`.